public/locales/en/panel-gettingstartedtour.json

```diff
@@ -0,0 +1,200 @@
+          "tip": "Open the navigation menu and click on the \"Focus\" button to focus on the Moon. The navigation menu can be found in the bottom bar"
+        }
+      },
+      "move-closer": {
+        "intro": "Now that we are focused on the Moon, let's move closer to it."
+      },
+      "conclusion": {
+        "intro": "Great! You have learned how to navigate in OpenSpace. You can now zoom in and out, rotate around objects, change focus, and look around freely.",
+        "next-step": "Now let's learn how to control time."
+      }
+    },
+    "time": {
+      "intro": {
+        "title": "Time",
+        "text": "In this chapter we will learn how to change the time in OpenSpace."
+      },
+      "view-solar-system": {
+        "intro": "But first, let's move to a good viewing point. Let's take a look at the Solar System from above."
+      },
+      "change-time": {
+        "intro": "Ok, now we can start changing the time. You can view the past as well as the future. Let's start by opening the Time Panel and changing the year.",
+        "task": {
+          "tip-time-panel": "To open the Time Panel, click on this icon in the taskbar",
+          "tip-year-input": "Change the year by changing the value of the year in the time input"
+        },
+        "hint-reset-time": "You can reset the time by pressing \"Now\""
+      },
+      "change-speed": {
+        "intro": " You can also change how fast time is playing. This is known as the <italic>simulation speed</italic> of the application, and the time increment is called the <italic>delta time</italic>.",
+        "task": {
+          "tip": "These buttons in the Time Panel adjust the delta time",
+          "rewind-aria-label": "Rewind time",
+          "forward-aria-label": "Fast forward time"
+        },
+        "hint-reset-speed": "You can reset the simulation speed by pressing \"Real Time\""
+      },
+      "pause-time": {
+        "intro": "You can also pause the time.",
+        "task": {
+          "tip": "In the Time Panel, click the pause button <pauseButton/> to pause time.to pause time."
+        },
+        "hint-speed-not-zero": "Note: simulation speed is not set to 0 when you pause."
+      },
+      "conclusion": {
+        "intro": "Fantastic! Now you can move around in both space and time.",
+        "next-step": "Let's look at some content!"
+      }
+    },
+    "content": {
+      "intro": {
+        "title": "Content",
+        "text": "In this chapter we will learn about the objects we can see in OpenSpace."
+      },
+      "scene-menu": {
+        "intro": "All the content in OpenSpace can be found in the Scene menu",
+        "search": "You can search for objects in the top search bar. The results are going to look like this",
+        "focus-info": "You can focus on objects by clicking the focus icon",
+        "focus-button-aria-label": "Focus object",
+        "checkbox-info": "To toggle the visibility, click on the checkbox next to the object name"
```

Comment:
```suggestion
        "checkbox-tooltip": "To toggle the visibility, click on the checkbox next to the object name"
```
? consistency

Reply:
Same here. not a tooltip
![image](https://github.com/user-attachments/assets/d4c7d6ba-ea8b-45c3-939f-e284ffe3d7cc)

I think that the getting started tour will have to be a bit inconsistent with the other files, just because the actual panel is very inconsistent/different to the others in terms of texts